use_cases/examples/sudden_appearance_new_dns_domain.conf

```diff
@@ -0,0 +1,24 @@
+##############################
+# SUDDEN APPEARANCE - NEW DNS DOMAIN
+##############################
+[sudden_appearance_new_dns_domain]
+es_query_filter=_exists_: Company
```

Comment:
this is very confusing - why is the target field, which I expect would be a domain name, called company?

Reply:
My bad, I made this use case for test purpose on my local machine.
I propose to switch the example use case to this one:
https://github.com/NVISO-BE/ee-outliers/blob/51e136e96c620ef841890a5a771c460a878eae35/use_cases/tests/sudden_appearance_new_process_location.conf#L1-L24
What do you think?